pkg/utils/urls.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package utils
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+)
+
+// ValidateURL takes in a url and returns a boolean if it's valid
+func ValidateURL(dst string) bool {
```

Comment:
All path handling is in `fpath` package.

`utils` package is usually the worst place where to put things because they end up as kitchen sinks.

Reply:
I'll move this over to fpath then.